drivers/dma/dma_mcux_lpc.c

```diff
@@ -19,14 +19,18 @@
 
 LOG_MODULE_REGISTER(dma_mcux_lpc, CONFIG_DMA_LOG_LEVEL);
 
+#define DMA_DESCRIPTOR_ALIGN(x, align)					\
```

Comment:
You can use `ROUND_UP` from `include/sys/util.h` instead

Reply:
Thank you, I will make this change